control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -717,6 +717,7 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
                     volume_context: HashMap::new(),
                     parameters: HashMap::new(),
                     volume_capabilities: caps,
+                    mutable_parameters: HashMap::new(),
```

Comment:
Presumably the `mutable_parameters` in the request are safe to ignore because we haven't reported we support the feature?

Reply:
Yep.